packages/renderer/src/lib/ui/Dropdown.svelte

```diff
@@ -0,0 +1,209 @@
+  if (opened && highlightIndex >= 0) {
+    onSelect(e, options[highlightIndex].value);
+    e.stopPropagation();
+  } else {
+    close();
+  }
+}
+
+function onEnter(i: number): void {
+  highlightIndex = i;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function onSelect(e: Event, newValue: any) {
+  onInput(newValue);
+  value = newValue;
+  close();
+  e.preventDefault();
+}
+
+function toggleOpen(): void {
+  if (opened) {
+    close();
+  } else {
+    open();
+  }
+}
+
+function open(): void {
+  opened = true;
+  highlightIndex = 0;
+}
+
+function close(): void {
+  opened = false;
+}
+
+function buildOptions(node: HTMLSelectElement) {
+  options = [...node.options].map(o => ({ value: o.value, label: o.textContent ?? '' }));
+  return {
+    destroy() {
+      options = [];
+    },
+  };
+}
+
+function onWindowClick(e: Event): void {
+  if (opened && e.target instanceof Node && !comp.contains(e.target)) {
+    close();
+  }
+}
+</script>
+
+<svelte:window on:click={onWindowClick} />
+
+<div
+  class="flex flex-row grow items-center px-1 py-1 group bg-[var(--pd-input-field-bg)] border-[1px] border-transparent min-w-24 relative {$$props.class ||
+    ''}"
+  class:not(focus-within):hover:bg-[var(--pd-input-field-hover-bg)]={!disabled}
+  class:focus-within:bg-[var(--pd-input-field-focused-bg)]={!disabled}
+  class:focus-within:rounded-md={!disabled}
+  class:focus-within:border-[var(--pd-input-field-hover-stroke)]={!disabled}
+  class:border-b-[var(--pd-input-field-stroke)]={!disabled}
+  class:hover:border-b-[var(--pd-input-field-hover-stroke)]={!disabled}
+  class:border-b-[var(--pd-input-field-stroke-readonly)]={disabled}
+  aria-invalid={$$props['aria-invalid']}
+  aria-label={$$props['aria-label']}
+  bind:this={comp}>
+  <button
+    class="flex flex-row w-full outline-0 bg-[var(--pd-input-field-bg)] placeholder:text-[color:var(--pd-input-field-placeholder-text)] items-center text-start"
+    class:text-[color:var(--pd-input-field-focused-text)]={!disabled}
+    class:text-[color:var(--pd-input-field-disabled-text)]={disabled}
+    class:group-hover:bg-[var(--pd-input-field-hover-bg)]={!disabled}
+    class:group-focus-within:bg-[var(--pd-input-field-hover-bg)]={!disabled}
+    class:group-hover-placeholder:text-[color:var(--pd-input-field-placeholder-text)]={!disabled}
+    disabled={disabled}
+    id={id}
+    name={name}
+    on:click={toggleOpen}
+    on:keydown={onKeyDown}>
+    <span class="grow">{selectLabel}</span>
+    <div
+      class:text-[var(--pd-input-field-stroke)]={!disabled}
+      class:text-[var(--pd-input-field-disabled-text)]={!disabled}
+      class:group-hover:text-[var(--pd-input-field-hover-stroke)]={!disabled}>
+      <Fa icon={faCaretDown} />
+    </div>
+  </button>
+
+  {#if opened}
+    <div
+      class="absolute top-full right-0 z-10 w-full max-h-80 rounded bg-[var(--pd-dropdown-bg)] border-[var(--pd-input-field-hover-stroke)] border-[1px] overflow-y-auto whitespace-nowrap">
```

Comment:
Here we are using `rounder`, and above we are using `rounded-md`

https://github.com/containers/podman-desktop/blob/70d2e3604d71ab5b867f682e62af733856afafc7/packages/renderer/src/lib/ui/Dropdown.svelte#L156

![image](https://github.com/user-attachments/assets/07c68fde-b544-4451-984f-c568c52472a9)

And some component above are using `rounder-sm`

https://github.com/containers/podman-desktop/blob/70d2e3604d71ab5b867f682e62af733856afafc7/packages/renderer/src/lib/image/RunImage.svelte#L853-L854

Should we make them match ?

Reply:
Updated both to rounded-md, which is what design uses.